### DISCUSSION TITLE
How to display top-level doc categories with multilingual support on a custom page

### DISCUSSION THREAD

**Discussion by weypro** [facebook/docusaurus#10292] [Q&A]
I want to display the top-level category list of docs on a custom page. The information to be displayed includes the category name (with multi-language support) and the number of documents in each category.

I've tried using usePluginData and useAllDocsData, but these don't provide a complete solution, especially in terms of multi-language support and accurate category names.

For reference, I'm looking to achieve an effect similar to what can be seen on the OpenAI Help Center: https://help.openai.com/en/

Please help me find a way to achieve this in Docusaurus.

**Comment by slorber** [comment]
Custom pages can't really access docs data unfortunately because plugins are decoupled/sandboxed. We'll likely solve this problem in the future thanks to React Server Components (https://github.com/facebook/docusaurus/discussions/8758)

However docs can display a list of categories. However there's a limitation to solve for docs at the top-level of a sidebar, with a workaround provided here: https://github.com/facebook/docusaurus/discussions/9923#discussioncomment-8710872
